edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Mqtt/MqttProtocolHead.cs

```diff
@@ -30,9 +30,9 @@ public class MqttProtocolHead : IProtocolHead
         const int DefaultListenBacklogSize = 200; // connections allowed pending accept
         const int DefaultParentEventLoopCount = 1;
         const int DefaultMaxInboundMessageSize = 256 * 1024;
-        const int DefaultThreadCount = 200;
         const bool AutoRead = false;
 
+        readonly int defaultThreadCount = Environment.ProcessorCount * 2;
```

Comment:
I am not sure we have a scenario where this needs to be an environment variable. In which case we can get rid of this altogether and just use the Dotnetty default (which is the same).

Reply:
It might be nice to have it in our back pocket. I don't think there is any scenario where we would go higher than this but it might be nice to be able to go lower.

Reply:
sure..